lib/creditsafe/request/find_company.rb

```diff
@@ -35,15 +46,29 @@ def message
         }
       end
 
-      private
+      # rubocop:disable Style/CyclomaticComplexity, Metrics/AbcSize
```

Comment:
Agreed. You can only treat this kinds of metrics as guidelines, as proxies for complexity. In reality, it's not really a complex method (relative to the validation requirements we're implementing). Let's stick to this.